Zend/zend_compile.h

```diff
@@ -507,6 +507,7 @@ struct _zend_execute_data {
 #define ZEND_CALL_FAKE_CLOSURE       (1 << 23)
 #define ZEND_CALL_GENERATOR          (1 << 24)
 #define ZEND_CALL_DYNAMIC            (1 << 25)
+#define ZEND_CALL_SYNTAX_PIPE        (1u << 26u)
```

Comment:
These are runtime flags, you'll want to add a flag (with value 1) somewhere below https://github.com/php/php-src/blob/4f7baffef9c30ef9ba45514fc0317875afc1e54f/Zend/zend_compile.h#L872.

Reply:
The flag also needs to fit into 16 bit, which causes the build failure on CI.